pkg/recipes/driver/gitconfig_test.go

```diff
@@ -128,19 +126,12 @@ func TestUnsetGitConfigForDir(t *testing.T) {
 	}
 }
 
-func getSecretList() v20231001preview.SecretStoresClientListSecretsResponse {
-	secrets := v20231001preview.SecretStoresClientListSecretsResponse{
-		SecretStoreListSecretsResult: v20231001preview.SecretStoreListSecretsResult{
-			Data: map[string]*v20231001preview.SecretValueProperties{
-				"username": {
-					Value: to.Ptr("test-user"),
-				},
-				"pat": {
-					Value: to.Ptr("ghp_token"),
-				},
-			},
-		},
+func getSecretList() map[string]string {
+	secrets := map[string]string{
```

Comment:
Would we need to test multiple secrets?

Reply:
We will test for only single secret at this time in gitconfig_test.go. gitconfig.go is working with a single secret for private git repository. I'll defer to @vishwahiremat for more details.